### PR TITLE
concourse: create-cloudfoundry has env specific bg

### PR DESCRIPTION
**Pull request by tlwr**
What
----

We need a signed commit to unblock the pipeline, may as well be the one which merges this PR

We also need more ducks, theatres, and regional flags in our lives.

Furthermore, it is nice to visually distinguish identical pipelines environmentally

How to review
-------------

Read the bash

View the pictures

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: concourse/scripts/pipelines-cloudfoundry.sh [hunk 1]
```diff
@@ -67,6 +67,25 @@ elif [ "${DEPLOY_ENV}" = "stg-lon" ]; then
   deploy_env_tag_prefix="v[0-9]*" # this matches all tags created by release ci
 fi
 
+case $DEPLOY_ENV in
+prod-lon)
+  # Flag of Greater London
+  BACKGROUND_IMAGE_URL="https://upload.wikimedia.org/wikipedia/commons/2/25/Flag_of_Greater_London.svg"
```
Comment (AP-Hunt) [inline concourse/scripts/pipelines-cloudfoundry.sh:73]: I like that you've gone and searched out the real flags, but I honestly would have no idea which one I was looking at from that :( Could we use 🏴󠁧󠁢󠁥󠁮󠁧󠁿  and 🇮🇪  ?

Comment (46bit) [reply concourse/scripts/pipelines-cloudfoundry.sh]: I've replaced that with the City of London flag: 

Flag of the City of London

Comment (LeePorte) [reply concourse/scripts/pipelines-cloudfoundry.sh]: Could we not use https://secretldn.com/wp-content/uploads/2017/02/london-classic-iconic-red-phoneboxes.jpg for London and https://www.irishcentral.com/uploads/article/20853/cropped_st_patricks_day_shamrocks___getty.jpg?t=1583226320 for Dublin?

I dislike flags

Comment (chrisfarms) [reply concourse/scripts/pipelines-cloudfoundry.sh]: Some fantastic bike shedding potential here ... _follows and gets popcorn_ :popcorn: :smile:

Comment (46bit) [reply concourse/scripts/pipelines-cloudfoundry.sh]: Went with Lee's ideas, they're good

**Review by LeePorte — APPROVED** [review]

**Pull request closed without merging**